tests/__init__.py

```diff
@@ -35,3 +35,18 @@ def get_testdata(*paths):
     path = os.path.join(tests_datadir, *paths)
     with open(path, 'rb') as f:
         return f.read()
+
+
+# FIXME: delete after dropping py2 support
+# Monkey patch the unittest module to prevent the
+# DeprecationWarning about assertRaisesRegexp -> assertRaisesRegex
+import sys
+if sys.version_info[0] == 2:
```

Comment:
six.PY2

Reply:
I see this as a very disposable piece of code that should go away once we drop Py2 support. Any particular reason for using `six`? Should we change https://github.com/scrapy/scrapy/pull/3877/files#diff-44aa4a970c57972e05d57dba01077acaL4 too?